### PR TITLE
Updating license

### PR DESCRIPTION
**Pull request by llorca**
Updating license to clarify patent license similar to Palantir’s [AtlasDB open source project](https://github.com/palantir/atlasdb)

### CODE REVIEW

File: packages/table/test/resizableTests.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /**
  * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
- * Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
```
Comment (giladgray) [inline packages/table/test/resizableTests.tsx]: we should probably include these two files in each of the published NPM packages. then we probably don't need to include these links and can fit it on one line like it used to be.

**Comment by themadcreator** [comment]
IANAL, but doesn't the apache-2.0 license already have a patent grant clause?

File: packages/table/test/resizableTests.tsx (hunk 1 above)
Comment (themadcreator) [inline packages/table/test/resizableTests.tsx]: Shouldn't they already be? They're not in `.gitignore` and I don't see a `.npmignore`

**Comment by adidahiya** [comment]
@llorca what is this blocked on?

**Comment by llorca** [comment]
@adidahiya not blocked anymore, my bad. Could use a quick review though!

File: docs/blueprint-landing.css [hunk 2]
```diff
@@ -1067,7 +1069,9 @@ span.pt-icon::before {
     outline: none !important; }
 
 /** Copyright 2015 Palantir Technologies, Inc. All rights reserved.
-// Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0 */
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS */
```
Comment (giladgray) [inline docs/blueprint-landing.css]: let's please fix the comment format. either all `/** ... */` block comments or `//` on every line. i prefer the former as it's what we used to do.

**Pull request merged**